portal/portal-impl/impl/src/java/org/sakaiproject/portal/entityprovider/PortalEntityProvider.java

```diff
@@ -296,12 +295,8 @@ public ActionReturn getFormattedProfile(EntityReference ref, Map<String, Object>
 
 		StringWriter writer = new StringWriter();
 
-		try {
-			formattedProfileTemplate.merge(context, writer);
-			return new ActionReturn(Formats.UTF_8, Formats.HTML_MIME_TYPE, writer.toString());
-		} catch (IOException ioe) {
-			throw new EntityException("Failed to format profile.", ref.getReference());
-		}
+		formattedProfileTemplate.merge(context, writer);
```

Comment:
Hi @axxter99 , thanks for this contribution, is this change related with the Velocity upgrade?

Reply:
Hi @mpellicer 

1.6.4:
public void merge(Context context,
                  Writer writer)
           throws ResourceNotFoundException,
                  ParseErrorException,
                  MethodInvocationException,
                  IOException

1.7:
public void merge(Context context,
                  Writer writer)
           throws ResourceNotFoundException,
                  ParseErrorException,
                  MethodInvocationException
\=https://velocity.apache.org/engine/1.7/apidocs/org/apache/velocity/Template.html

Reply:
Sounds good!